superset/commands/dashboard/delete.py

```diff
@@ -37,6 +38,19 @@
 logger = logging.getLogger(__name__)
 
 
+class DeleteEmbeddedDashboardCommand(BaseCommand):
```

Comment:
Are we missing the usage of this command in some other place besides the test file?

Reply:
Oh wow, the commit did not include it. Thanks for this!